CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * [FEATURE] Add new metrics to `replication_group_member_stats` collector to support MySQL 8.x.
 * [FEATURE] Add collector for `replication_group_members` (PR #459) #362
 * [BUGFIX] Fixed output value of wsrep_cluster_status #473
+* [FEATURE] Add collector for `performance_schema.memory_summary_global_by_event_name` (PR #515) #75
```

Comment:
This should be sorted before BUGFIX lines.

Reply:
Actually, don't worry about it, I have some other cleanup to do in the CHANGELOG.